README.md

```diff
@@ -162,25 +162,47 @@ Here's an example of a basic ProfileUploader component, demonstrating how to add
 
 We currently re-expose withWebId and LogoutButton so you can use the basic components without installing other libraries.
 
-### ShExFormBuilder
+### FormModel
 
-**Note**: This is an early preview build of a work-in-progress component and subject to large-scale changes in future releases.
-
-This component allows you to create a fully functional form, complete with submit, cancel, and validation rules, from a [ShEx](http://shex.io/shex-primer/index.html) Shape.
+This component takes a FormModel (build using the Form Language and [UI ontology](https://www.w3.org/ns/ui#)). A properly formatted FormModel will render a form, complete with constraints. Some things are not yet supported, such as ui:arc links, FormForms, and more. This component is still being improved and worked on.
```

Comment:
It is not really that FormForm is not supposed, it is that ui:Choice and ui:Options are not supported.